action.yml

```diff
@@ -83,15 +91,58 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Log the next step action
+    run: echo ▷ Figuring out the environment Python version...
+    shell: bash
+  - name: Compute job Python version
+    id: compute-origin-python-version
+    run: |
+      # Compute job Python version
+      import sys
+
+      # Input from GHA
+      origin_python_version = '${{ inputs.origin-python-version }}'
+      old_python_version = '${{ inputs.python-version }}'
+      ansible_core_version = '${{ inputs.ansible-core-version }}'
+
+      # Handle case that deprecated Python version parameter is provided
+      if old_python_version:
+          # Check whether the default of origin-python-version was changed
+          if origin_python_version != 'auto':
+              sys.exit(
+                  '::error ::`python-version` and `origin-python-version` cannot'
```

Comment:
@felixfontein are you sure that combining this would work? https://docs.python.org/3/library/sys.html#sys.exit says that it'll be printed to `stderr` but https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#using-workflow-commands-to-access-toolkit-functions only mentions `stdout` as a target for workflow commands.